pom.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2016 Cask Data, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.planningworks.cdap.plugins</groupId>
+  <artifactId>anaplan.plugins</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Anaplan Plugins</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <cdap.version>6.2.3</cdap.version>
+    <cdap.plugin.version>2.6.0-SNAPSHOT</cdap.plugin.version>
+    <etl.versionRange>[6.0.0,10.0.0-SNAPSHOT)</etl.versionRange>
+    <exported-packages>com.google.planningworks.cdap.plugins.*;</exported-packages>
+
+    <!--  Anaplan properties below  -->
+    <spark.version>1.6.1</spark.version>
+    <feign-jackson.version>10.0.0</feign-jackson.version>
```

Comment:
nit: can you arrange these alphabetically? Makes it easier to find a specific version.

Reply:
Thanks, update!